gazebo_plugins/src/gazebo_ros_bumper.cpp

```diff
@@ -114,6 +130,12 @@ void GazeboRosBumper::Load(sensors::SensorPtr _parent, sdf::ElementPtr _sdf)
   this->contact_pub_ = this->rosnode_->advertise<gazebo_msgs::ContactsState>(
     std::string(this->bumper_topic_name_), 1);
 
+  this->locus_bump_pub_ = this->rosnode_->advertise<std_msgs::Bool>(
+      std::string(this->locus_bump_topic_name_), 1, true);
+  // Initial false bump report - if this is incorrect it will be replaced
```

Comment:
Technically, this could break from the physical robot. Our power-management node won't publish to `bump` until it hears from the MCB, which will theoretically tell it `bump` is true if the bumper is engaged on start. However, this will only cause issues if we restart the gazebo plugin without restarting the rest of the simulated robot. In that case it might temporarily unpause the robot before the next publish cycle.